### ISSUE TITLE
Refactor: Variable prompt:"" for API calls to OpenAI for Reusable Function 

### ISSUE THREAD

**Issue by ash1eygrace** [ash1eygrace/ai-content#4]
This refactor is dependent on completion of #3. 

Simple example request to OpenAI:

```
const response = await openai.createCompletion("text-davinci-002", {
  prompt: "",
  temperature: 0.7,
  max_tokens: 256,
  top_p: 1,
  frequency_penalty: 0,
  presence_penalty: 0,
});
```
Currently in each [component](https://github.com/ash1eygrace/ai-content/tree/main/copyai/src/components) the prompt includes a string with a variable depending on the form response i.e: [Company Bio](https://github.com/ash1eygrace/ai-content/blob/6a4d046623ffd9b87afa01e2e7ae1485bf52b773/copyai/src/components/CompanyBio.js#L31) shows:

```
prompt: `Write a persuasive and exciting company bio for: ${formDataObj.companyName}`,
```

For refactor, the entire prompt needs to be a variable with the components form submission variable`${formDataObj.companyName}` being passed into that variable i.e. initial prompt per [component](https://github.com/ash1eygrace/ai-content/tree/main/copyai/src/components): 

```
prompt: `Write a persuasive and exciting company bio for: ${formDataObj.companyName}`,
const companyNamePrompt = prompt. 
```

Final prompt from the component sent to the [reusable function](https://github.com/ash1eygrace/ai-content/issues/3) is then:

```
const response = await openai.createCompletion("text-davinci-002", {
prompt: companyNamePrompt,
```

Final prompt to OpenAI send from the reusable function :

```
const response = await openai.createCompletion("text-davinci-002", {
  prompt: companyNamePrompt,
  temperature: 0.7,
  max_tokens: 256,
  top_p: 1,
  frequency_penalty: 0,
  presence_penalty: 0,
});
```

**Comment by ash1eygrace** [comment]
Planned structure (still needs research):
```
├── components
│   ├── pages
│   │   ├── about.js
│   │   ├── contact.js
│   │   └── Home.js
│   ├── API
│   │   └── reusable-function.js
│   ├── forms
│   │   ├── BlogIntro.js
│   │   ├── CompanyBio.js
│   │   ├── LinkedInJobDescription.js
│   │   ├── Navigation.js
│   │   └── ProductDescription.js
```

**Comment by ash1eygrace** [comment]
Solved in Refactor api function #15
